README.md

```diff
@@ -53,10 +53,10 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
         #Activate the environment
         #Depending on your version of Anaconda use one of the following:
 
-        #Anaconda 3.4 and up:
+        #Anaconda 3.4 and up (conda>=4.6):
         conda activate isis
 
-        #Prior to Anaconda 3.4:
+        #Prior to Anaconda 3.4 (conda<4.6):
```

Comment:
I'm confused here, are you adding these parenthetical statements because the `source activate` command is still allowable for conda until conda 4.6?  If so, I wouldn't include this.  It doesn't particularly clarify the instructions for novices, and conda experts like yourself already know these historical caveats.

Reply:
I was thinking of miniconda users that don't know Anaconda versions at all.

Reply:
Oh ... Anaconda and miniconda are not version-locked? Did not know. In that case, I'd explicitly say "miniconda" instead of "conda" in your parenthetical. "conda" is often used as the generic thing, and could mean either (and that's how I was parsing it!)